packit_service/service/api/testing_farm.py

```diff
@@ -134,3 +137,40 @@ def validate_testing_farm_request():
         msg = "Invalid testing farm secret provided"
         logger.warning(msg)
         raise ValidationFailed(msg)
+
+
+@ns.route("/get-results")
+class TestingFarmResultsList(Resource):
+    @ns.expect(pagination_arguments)
+    @ns.response(HTTPStatus.PARTIAL_CONTENT, "Testing Farm Results list follows")
+    def get(self):
```

Comment:
I think it would make more sense to make this part of `TestingFarmResults`, instead of having a separate route for it.

Reply:
TestingFarmResults is a POST webhook at /testing-farm/results

Its not mentioned here but /get-results means /testing-farm/get-results

Reply:
Yeah, I know that, but usually in a REST API one can use the same endpoint to both save data in the service or retrieve it, and this is the effect that we need here: POST to /testing-farm/results will save results, GET to /testing-farm/results will list them.

Reply:
Oh, sorry. I thought you meant something else.